spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionController.java

```diff
@@ -215,6 +216,12 @@ public TaskExecutionResource view(@PathVariable("id") long id) {
 		return resources;
 	}
 
+	@RequestMapping(method = RequestMethod.GET, params = "count")
+	@ResponseStatus(HttpStatus.OK)
+	public TaskExecutionsResource getTaskExecutionIds(@RequestParam(defaultValue = "false", name="completed") boolean completed) {
```

Comment:
This also has other issues like:
```
http ":9393/tasks/executions?name=t1&count"

HTTP/1.1 500 
Connection: close
Content-Type: application/hal+json
Date: Tue, 23 Feb 2021 09:08:03 GMT
Transfer-Encoding: chunked

{
    "_embedded": {
        "errors": [
            {
                "_links": {
                    "self": {
                        "href": "/"
                    }
                },
                "logref": "IllegalStateException",
                "message": "Ambiguous handler methods mapped for '/tasks/executions': {public org.springframework.hateoas.PagedModel org.springframework.cloud.dataflow.server.controller.TaskExecutionController.retrieveTasksByName(java.lang.String,org.springframework.data.domain.Pageable,org.springframework.data.web.PagedResourcesAssembler), public org.springframework.cloud.dataflow.rest.resource.TaskExecutionsResource org.springframework.cloud.dataflow.server.controller.TaskExecutionController.getTaskExecutionIds(boolean)}"
            }
        ]
    }
}

```

Reply:
Also these both gives same response:
```
http ":9393/tasks/executions?count&completed=true"
http ":9393/tasks/executions?count&completed=false"
```

Reply:
Ah it's inclusion, so gives different count when things are running.

Reply:
Yeah, I see the ambiguity and will fix it.